common/scala/src/main/resources/application.conf

```diff
@@ -19,7 +19,11 @@ akka.http {
 
 #kamon related configuration
 kamon {
-
+    environment {
+      # Identifier for this service. For keeping it backward compatible setting to natch previous
+      # statsd name
+      service = "openwhisk-statsd"
```

Comment:
Setting this to `openwhisk-statsd` for ensuring that new metric names are backward compatible with previous names as now reporters use this tag to determine the service name and do not expose reporter specific setting